flake.nix

```diff
@@ -57,5 +57,86 @@
       packages = {
         inherit (pkgs) nixUnstable;
       };
+
+      # nix flake check
+      checks = let
```

Comment:
Let's somehow mention that these tests are supposed to test `wrappendNix` from the overlay

Reply:
I was wondering if there might be a better place for these checks

Reply:
I think you can probably extract this wrapper in a dedicated flake, just like you did in https://github.com/serokell/ops-1339-wrapper/blob/main/flake.nix and re-expose checks from this flake in these top-level checks